HW15/src/FileLogger.java

```diff
@@ -0,0 +1,32 @@
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class FileLogger {
+
+    private final FileLoggerConfiguration fileLoggerConfiguration;
+
+
+    public FileLogger(){
+        fileLoggerConfiguration = new FileLoggerConfiguration();
+    }
+
+
+    public void info(String message) throws IOException, FileMaxSizeReachedException {
+        if(!fileLoggerConfiguration.isMaxFileSizeExceeded()) {
```

Comment:
According to requirement 6 - you have to write to log anyway (and create a new file for logs).